spring-xsuaa/pom.xml

```diff
@@ -42,7 +42,7 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-autoconfigure</artifactId>
 			<version>${spring.boot.version}</version>
-			<scope>provided</scope>
+			<optional>true</optional>
```

Comment:
Hm...this adds a dependency to a Spring version.  This implies every time Spring updates a new version, this lib also needs to update. Why is this needed?